kong/tools/queue_schema.lua

```diff
@@ -0,0 +1,46 @@
+local Schema = require "kong.db.schema"
+
+-- TODO: enable the descriptions once they are accepted in Schemas
+return Schema.define {
+  type = "record",
+  fields = {
+    { name = {
+      type = "string",
+      -- description = "name of the queue, unique across one workspace.",
+      -- If two plugin instances use the same queue name, they will
+      -- share one queue and their queue related configuration must match.
```

Comment:
What is the behavior when the name is not set by the end-user?
(asking in terms of end-user behavior, not the behavior to be expected by a lua developer)